bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```
@@ -647,4 +647,4 @@ spec:
   provider:
     name: MongoDB, Inc
   version: 2.3.0
-  replaces: mongodb-atlas-kubernetes.v2.2.2
+  replaces: mongodb-atlas-kubernetes.v2.3.0
```

We need to fix this reverting one:
```suggestion
  replaces: mongodb-atlas-kubernetes.v2.2.2
```